src/pages/TravelTipsPage/TravelTipsAllCards.jsx

```diff
@@ -0,0 +1,74 @@
+import TravelTipsBtn from '../../components/common/Buttons/TravelTipsButtons/TravelTipsBtn';
+import SearchPanelTravelTips from '../../components/common/SearchPanels/SearchPanelTravelTips';
+import TravelTipsCard from '../../components/common/TravelTipsCard/TravelTipsCard';
+import tipsData from '../../data/tipsData';
+import { useState, memo } from 'react';
+
+const TravelTipsAllCards = () => {
+  const [activeButtons, setActiveButtons] = useState([]);
+
+  const travelTipsButtons = [
+    { text: 'Packing' },
+    { text: 'Transportation' },
+    { text: 'Accommodation' },
+    { text: 'Local Cuisine' },
+    { text: 'Cheap Trip' },
+    { text: 'All Tips' },
```

Comment:
Usually, it is not safe to rely on text that you are showing to the user. Things like this should have key and value. It is gonna look like this.

`{ key: 'ALL', text: 'All tips' }`

This is because text displayed to user might change in some time, and might be translated in contrast to key that shouldn't get translated or changed

Reply:
The change will affect the `tipsData` array as well. The category field there should be an object with the structure I was mentioned above.